Dockerfile-cuDNN

```diff
@@ -36,11 +36,13 @@ RUN unset LD_LIBRARY_PATH \
     && echo "deb ${NVIDIA_URL}/machine-learning/repos/${NVIDIA_DISTRO}/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list \
     && apt-get update -yq --fix-missing \
     && apt-get install -yq --no-install-recommends \
-       "cuda-cudart-${NVIDIA_CUDA_MAJOR_VERSION}=${NVIDIA_CUDA_PKG_VERSION}" \
-       "cuda-libraries-${NVIDIA_CUDA_MAJOR_VERSION}=${NVIDIA_CUDA_PKG_VERSION}" \
-       "cuda-libraries-dev-${NVIDIA_CUDA_MAJOR_VERSION}=${NVIDIA_CUDA_PKG_VERSION}" \
+       "cuda-cudart-${NVIDIA_CUDA_MAJOR_VERSION}=${NVIDIA_CUDA_PKG_VERSION}-1" \
```

Comment:
Shouldn't we make `1` into a MINOR version env variable?

Reply:
I was just aligning to Nvidia's mechanism in their [Dockerfile](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/base/Dockerfile) but we could explicitly add a `MINOR` if you still wish it :)